.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -360,23 +398,32 @@ main() {
     NAME_SPACE="showcase-ci-nightly"
     NAME_SPACE_RBAC="showcase-rbac-nightly"
     NAME_SPACE_POSTGRES_DB="postgress-external-db-nightly"
+    NAME_SPACE_AKS="showcase-aks-ci-nightly"
   fi
 
   install_oc
-  oc login --token="${K8S_CLUSTER_TOKEN}" --server="${K8S_CLUSTER_URL}"
+  oc login --token="${K8S_CLUSTER_TOKEN}" --server="${K8S_CLUSTER_URL}" --insecure-skip-tls-verify
   echo "OCP version: $(oc version)"
 
   API_SERVER_URL=$(oc whoami --show-server)
-  K8S_CLUSTER_ROUTER_BASE=$(oc get route console -n openshift-console -o=jsonpath='{.spec.host}' | sed 's/^[^.]*\.//')
+  if [[ "$JOB_NAME" == *aks* ]]; then
+    K8S_CLUSTER_ROUTER_BASE=$(kubectl get svc nginx --namespace app-routing-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
```

Comment:
what if there is no nginx running, I would fail here.

Reply:
I agree that it would be better to make sure it runs and enable it if it's not. As it is now, enabling it would be a [manual step](https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/main/install-rhdh-aks/#proc-rhdh-deploy-aks-helm_assembly-install-rhdh-aks) for setting up the cluster before using it for running tests.

Reply:
I added this to make sure it is enabled: https://github.com/janus-idp/backstage-showcase/pull/1472/files#diff-4d6549ba4c59a94aab379c14838198870ef2e610c26348b6c58c6fbffedc479bR392